src/rating-indicator.scss

```diff
@@ -176,37 +178,33 @@ $block: #{$fd-namespace}-rating-indicator;
       @include fd-icon-glyph('favorite') {
         color: var(--fdRating_Indicator_Disabled_Selected_Color);
       }
+    }
 
-      &:checked ~ input + .#{$block}__label {
-        @include fd-icon-glyph('favorite') {
-          color: var(--fdRating_Indicator_Disabled_Unselected_Color);
-        }
+    @include fd-rating-indicator-labels-after-checked() {
+      @include fd-icon-glyph('favorite') {
+        color: var(--fdRating_Indicator_Disabled_Unselected_Color);
       }
     }
   }
 
-  @include fd-flex-center();
-
-  &.is-display-mode {
+  &.is-display-mode,
+  &.is-non-interactive {
```

Comment:
this needs to be real modifier classes:
`fd-rating-indicator fd-rating-indicator--non-interactive`
`fd-rating-indicator fd-rating-indicator--display-mode`

Reply:
I know `is-display-mode` was added before this PR but it's just something we didn't see in the prev review. You will need to add it as a Breaking change.

Reply:
Oh, okay.

I thought not to break the approach, but okay, let's change it!